spec/support/importer.rb

```diff
@@ -23,3 +23,16 @@
     importer.import
   end
 end
+
+shared_examples 'a versionable resource' do
+  before { described_class.model_class.recreate_index }
+
+  it 'updates version properly' do
+    expect(importer.stored_version).not_to eq importer.available_version
+    importer.import
+
+    # When resource is unchanged, stored_version should be equal available_version
+    new_importer = importer.class.new(resource)
+    expect(new_importer.stored_version).to eq new_importer.available_version
+  end
+end
```

Comment:
Also, can you rebase this branch onto master to pick up [these changes](https://github.com/GovWizely/webservices/pull/207)? They are somewhat related to the work in this PR.